index.html

```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
           </div>
         </div>
         </div> 
-      </section>
-    
+    </section>
+
+    <section class="contact-container">
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required maxlength="30" placeholder="Text Input">
+        <input type="email" name="email" required maxlength="30" id="email"  placeholder="test@gmail.com">
```

Comment:
- [ ] [OPTIONAL] Kindly remove the max length in your email field. It's not a requirement and it limits the number of characters in email, the best practice is to provide the format for the email input, which is already taken of by ` type="email"`.